kasa/feature.py

```diff
@@ -231,6 +231,8 @@ def value(self) -> int | float | bool | str | Enum | None:
         """Return the current value."""
         if self.type == Feature.Type.Action:
             return "<Action>"
+        if self.type is Feature.Type.Number and not self.attribute_getter:
+            return 0
```

Comment:
If this is necessary for write-only features, I think we need to find a nicer solution as having something like this can cause issues that are hard to debug.

Reply:
Ok have implemented this with a default step so pan_left/tilt_up etc is a normal action. We can look into allowing chaining cli feature commands so multiple can be sent in one go which would be nice.